### DISCUSSION TITLE
Run the GUI and CLI at same time?

### DISCUSSION THREAD

**Discussion by Alister-Zeus** [qarmin/czkawka#1127] [Q&A] [answered]
I'm doing a search of both images and videos

It seems silly to have the CPU sitting _idle_ when I'm checking the images for duplicates, when it could be filling the cache with the videos

Is this possible or advised?

**Comment by qarmin** [comment, marked as answer]
Nowhere in the manual I wrote this, but it would be useful to add that it is not recommended to use several app instances at once using the same cache files, because the action in this case is not defined(cache file can reset, delete, the application can shut down).

Searching for similar images/videos in two different instances should work correctly, because they use different cache files

**Comment by Alister-Zeus** [comment]
Thanks I noticed they used different cache files, hence my question

I keep trying to find an ideal duplicate finder, that can make maximum use of the systems I have, if I ever win the the lottery I'm paying someone to write it.....
